### PR TITLE
Adding new navbar (using ember-styleguide) and footer

### PR DESCRIPTION
**Pull request by mansona**
Wow, what an amazing day 🎉 I am completely surprised what we have been able to achieve on this PR and I feel incredibly grateful being able to work with such awesome people 😍 

This is now ready for review, so let's get this merged 💪

### CODE REVIEW

File: app/styles/app.scss [hunk 1]
```diff
@@ -5028,3 +5028,5 @@ body.tomster_faq #subcontent #back-to-top {
 .screenshot {
   margin-left: -60px;
 }
+
+@import "ember-bootstrap/bootstrap";
```
Comment (rwwagner90) [inline app/styles/app.scss]: Please remove this, as the necessary styles are already included in styleguide.

File: package.json [hunk 2]
```diff
@@ -18,9 +18,11 @@
     "deploy": "ember deploy production"
   },
   "devDependencies": {
+    "bootstrap-sass": "^3.3.7",
```
Comment (rwwagner90) [inline package.json]: Should be able to remove both bootstrap deps from package.json

**Comment by rwwagner90** [comment]
We still have some styles to fix for the navbar, but functionally, this should be good to go.

**Comment by serenaf** [comment]
Thanks so much @mansona @rwwagner90 ❤️ You guys rock!

**Pull request merged**